superduperdb/base/serializable.py

```diff
@@ -108,6 +108,9 @@ def dict(self):
 
         return Document(asdict(self))
 
+    def copy(self):
+        return self.decode(self.encode())
```

Comment:
can we directly copy `self`

Reply:
I agree `copy.deepcopy`?

Reply:
```
from copy import copy, deepcopy

from superduperdb.backends.mongodb import Collection

deepcopy(Collection('test').find())
copy(Collection('test').find())
```

I have tried it before, but it seems that copy and deepcopy are incompatible with some of our implementations, which will cause the variables copied to be referenced recursively. Haven’t checked the problem carefully yet

Reply:
Already fixed